src/applications/personalization/profile/components/direct-deposit/DirectDeposit.jsx

```diff
@@ -8,8 +8,8 @@ import { focusElement } from 'platform/utilities/ui';
 import { handleDowntimeForSection } from '../alerts/DowntimeBanner';
 
 import FraudVictimAlert from './FraudVictimAlert';
-import DirectDepositCNP from './DirectDepositCNP';
-import DirectDepositEDU from './DirectDepositEDU';
+import BankInfoCNP from './BankInfoCNP';
+import DirectDepositEDU from './DirectDepositEDUEbenefits';
 
 const DirectDeposit = () => {
```

Comment:
Can we rename this file to DirectDepositV1?

Reply:
done!